pkgs/tools/graphics/vulkan-cts/default.nix

```diff
@@ -0,0 +1,147 @@
+{ lib, stdenv
+, fetchFromGitHub
+, fetchurl
+, cmake
+, libdrm
+, libglvnd
+, libffi
+, libpng
+, libX11
+, libXau
+, libXdmcp
+, libxcb
+, makeWrapper
+, ninja
+, pkg-config
+, python3
+, vulkan-loader
+, wayland
+, wayland-protocols
+, zlib
+}:
+let
+  renderdoc = fetchurl {
+    url = "https://raw.githubusercontent.com/baldurk/renderdoc/v1.1/renderdoc/api/app/renderdoc_app.h";
+    hash = "sha256-57XwqlsbDq3GOhxiTAyn9a8TOqhX1qQnGw7z0L22ho4=";
+  };
+
+  # The build system expects all these dependencies inside the external folder and
+  # does not search for system-wide installations.
+  # It also expects the version specified in the repository, which can be incompatible
+  # with the version in nixpkgs (e.g. for SPIRV-Headers), so we don't want to patch in our packages.
```

Comment:
Then we might as well fetch them via the submodules to reduce maintenance overhead.

Reply:
I’d like to do that but these dependencies are not submodules.
They are fetched by a python script that hardcodes the hashes (and I copied them from there), see the first few lines of the build instructions: https://github.com/KhronosGroup/VK-GL-CTS/blob/main/external/vulkancts/README.md#building-cts

Reply:
oh well...